app/src/main/java/com/goldenraven/padawanwallet/ui/tutorials/TutorialsHomeScreen.kt

```diff
@@ -0,0 +1,86 @@
+package com.goldenraven.padawanwallet.ui.tutorials
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import com.goldenraven.padawanwallet.R
+import com.goldenraven.padawanwallet.theme.md_theme_dark_background
+import com.goldenraven.padawanwallet.ui.Screen
+
+@OptIn(ExperimentalMaterial3Api::class, androidx.compose.animation.ExperimentalAnimationApi::class)
+@Composable
+internal fun TutorialsHomeScreen(navController: NavController) {
+
+    val tutorialsNameList = arrayListOf(
+        listOf(stringResource(R.string.E1_title), stringResource(R.string.concept)),
+        listOf(stringResource(R.string.E2_title), stringResource(R.string.concept)),
+        listOf(stringResource(R.string.E3_title), stringResource(R.string.skill)),
+        listOf(stringResource(R.string.E4_title), stringResource(R.string.concept)),
+        listOf(stringResource(R.string.E5_title), stringResource(R.string.skill)),
+        listOf(stringResource(R.string.E6_title), stringResource(R.string.concept)),
+        listOf(stringResource(R.string.E7_title), stringResource(R.string.concept)),
+        listOf(stringResource(R.string.E8_title), stringResource(R.string.skill))
+    )
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(md_theme_dark_background)
+    ) {
+        LazyColumn {
+            itemsIndexed(tutorialsNameList) { index, item ->
+                Card(
+                    shape = RoundedCornerShape(8.dp),
+                    modifier = Modifier
```

Comment:
I'm using the Material3 framework because I wanted to force myself to learn it and thought this was good project for it, and overall I think it's ok, but one of the things I really dislike is how it messes with your colours. On some items it adds some sort of filter depending on the elevation of the composable and whatnot, and just _creates_ new colours out of thin air... it's not great. In this case this is how we end up with the weird sort of purple that the cards are. It's not defined anywhere it's just applied on the spot. Anyway let's use one of the Padawan colours for now if you don't mind. It's not perfect (feel free to propose another one if you really don't like my pick), but let's try this instead:
```kotlin
Card(
    shape = RoundedCornerShape(8.dp),
    containerColor = md_theme_dark_background2,
    modifier = Modifier
        .padding(16.dp)
        .fillMaxWidth()
        .clickable(onClick = {
            navController.navigate(Screen.TutorialsScreen.route + "/$index")
        })
) 
```